packages/format-library/src/link/modal-screens/link-picker-screen.native.js

```diff
@@ -20,31 +20,20 @@ const LinkPickerScreen = () => {
 	const navigation = useNavigation();
 	const route = useRoute();
 	const onLinkPicked = ( { url, title } ) => {
-		if ( Platform.OS === 'android' ) {
-			Keyboard.dismiss();
-			delay( () => {
-				navigation.navigate( linkSettingsScreens.settings, {
-					inputValue: url,
-					text: title,
-				} );
-			}, 100 );
-			return;
-		}
```

Comment:
My presumption is that this conditional logic was placed to improve animations on Android at some point. It would similar logic is now beneficial for iOS as well.